src/ClusterBootstrap/install_prerequisites.sh

```diff
@@ -1,42 +1,42 @@
-#!/bin/bash 
-
-sudo apt-get update 
-sudo apt-get install -y --no-install-recommends \
-        apt-utils \
-        software-properties-common \
-        git \
-        curl \
-        python-pip \
-        wget \
-        cpio \
-        mkisofs \
-        apt-transport-https \
-        openssh-client \
-        ca-certificates 
-
-# Install docker
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-
-sudo add-apt-repository \
-   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
-   $(lsb_release -cs) \
-   stable"
-    docker-ce
-
-AZ_REPO=$(lsb_release -cs)
-echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $AZ_REPO main" | \
-    sudo tee /etc/apt/sources.list.d/azure-cli.list
-
-sudo apt-key --keyring /etc/apt/trusted.gpg.d/Microsoft.gpg adv \
-     --keyserver packages.microsoft.com \
-     --recv-keys BC528686B50D79E339D3721CEB3E94ADBE1229CF
-
-sudo apt-get update
-sudo apt-get install -y --no-install-recommends \
-sudo apt-get install azure-cli
-
-pip install --upgrade pip
-pip install setuptools && pip install pyyaml && pip install jinja2
-
-sudo echo "dockerd > /dev/null 2>&1 &" | cat >> /etc/bash.bashrc
-
+#!/bin/bash 
+sudo apt-get update 
+sudo apt-get install -y --no-install-recommends \
+        apt-utils \
+        software-properties-common \
+        git \
+        curl \
+        python-dev \
+        python-pip \
+        wget \
+        cpio \
+        mkisofs \
+        apt-transport-https \
+        openssh-client \
+        ca-certificates \
+        network-manager

```

Comment:
why do we need network-manager?

Reply:
this is on devbox, one time effort. it's once used to edit /etc/resolv.conf, I'd test whether now we can get rid of it.